test/test_cuda.py

```diff
@@ -3139,6 +3148,81 @@ def forward(self, x):
             model_graphed({"x": real_inputs[0]}), model_control({"x": real_inputs[0]})
         )
 
+    @unittest.skipIf(
+        not TEST_CUDA_GRAPH, "CUDA >= 11.0 or ROCM >= 5.3 required for graphs"
```

Comment:
We can remove the CUDA 11.0 text now as there are no CUDA < 11.8 builds on upstream

Reply:
Should that be removed for everything in this file? If so, I am not sure how to handle the following line because the error message only mentions CUDA version, but the skip condition looks like it is checking for more than just that...

https://github.com/pytorch/pytorch/blob/1757337b63160a8113ac73d8439c290e033b3d9c/test/test_cuda.py#L2305-L2306